service/src/utils/cli_parsers/parseRegisterArgs.ts

```diff
@@ -0,0 +1,95 @@
+import { ethers } from "ethers";
+import { exit } from "process";
+import { Web3Interface } from "scripts/web3-interface";
+import { archLogger } from "../chalk-theme";
+import { CLI_BAD_REGISTER_PROFILE_ARG } from "../exit-codes";
+
+const diggingFee = 'digging-fee';
+const rewrapInterval = 'rewrap-interval';
+const freeBond = 'free-bond';
+
+export async function parseRegisterArgs(web3Interface: Web3Interface): Promise<{
+    minimumDiggingFee: ethers.BigNumber;
+    maximumRewrapInterval: number;
+    freeBond: ethers.BigNumber;
+}> {
+    const argsStr = process.argv.toString().split("--")[1];
+
+    if (!argsStr) {
+        archLogger.error("Missing arguments to register");
+        exit(CLI_BAD_REGISTER_PROFILE_ARG);
+    }
+
+    const args = argsStr.split(",").map(a => a.trim()).filter(a => a !== "");
+
+    const processedArgs: string[] = [];
+
+    const registerParams = {
+        diggingFee: ethers.constants.Zero,
+        rewrapInterval: 0,
+        freeBond: ethers.constants.Zero,
+    }
```

Comment:
This could potentially be defined as an interface for type checking (not sure if my types below are correct):

```
interface ProfileArgs {
  diggingFee: BigNumber | string,
  rewrapInterval: number | string,
  freeBond: BigNumber | string
}
```